pkg/controller/composite/controller.go

```diff
@@ -304,6 +316,14 @@ func (pc *parentController) processNextWorkItem() bool {
 }
 
 func (pc *parentController) enqueueParentObject(obj interface{}) {
+	// If the parent doesn't match our selector, and it doesn't have our
```

Comment:
query: I am unsure of this. Hence asking. 
Is it fine to enqueue if:
1/ cond1=true _i.e. parent does not contain finalizer_ & cond2=false _i.e. parent labels match_ -> leading to enqueue
2/ cond1=false _i.e. parent contains finalizer_ & cond2=true _i.e. parent labels don't match_ -> leading to enqueue

Reply:
I was basin on https://github.com/metacontroller/metacontroller/blob/master/pkg/controller/decorator/controller.go#L315-L320

re.1 it is true as it could be new object, so we might  add finalizer
re.2 it it is true, it means that i.e. object loose its labels and we might have some work to do during releasing the resource


at least up to my knowledge ;)

Reply:
Thanks!!!